src/common/timer/interaction-timer.js

```diff
@@ -37,11 +47,29 @@ export class InteractionTimer extends Timer {
     }
   }
 
+  clear (abort) {
+    clearTimeout(this.timer)
+    this.timer = null
```

Comment:
```suggestion
    super.clear()
```

Reply:
Better to either:

1.  rename this to `abort` and let this class' refresh continue to use base Timer's clear but session-entity's reset uses this `abort`, OR
2. make the abort behavior in this `clear` mandatory, and use `super.clear()` in this class' refresh.

Reply:
good call